charts/reporting-operator/templates/custom-resources/report-queries/persistentvolumeclaim-status.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "persistentvolumeclaim-status-raw"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
+  - "persistentvolumeclaim-status"
+  columns:
+  - name: namespace
+    type: string
+  - name: persistentvolumeclaim
+    type: string
+  - name: labels
+    type: map<string, string>
+    tableHidden: true
+  - name: bound
+    type: double
+  - name: status
+    type: string
+  - name: timeprecision
+    type: double
+    unit: seconds
+  - name: timestamp
+    type: timestamp
+    unit: date
+  - name: dt
+    type: string
+  query: |
+    SELECT
+        labels['namespace'] as namespace,
+        labels['persistentvolumeclaim'] as persistentvolumeclaim,
+        labels,
+        amount as bound,
```

Comment:
Since `phase` is what indicates the state, and I believe the value is always 1, calling this `bound` doesn't make sense. I think we can probably just call it `value` or something simple.

Reply:
done